hands-on-exercise/3-cosmjs-adv/1-cosmjs-objects.md

```diff
@@ -165,20 +107,84 @@ $ mkdir -p proto/gogoproto
 $ curl https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.45.4/third_party/proto/gogoproto/gogo.proto -o proto/gogoproto/gogo.proto
 ```
 
-Now compile:
+Now we're ready to compile:
+
+Create a telescope config `.telescope.json` with following content:
```

Comment:
I think it would be worth adding a note after this JSON pointing out important elements. I am thinking of `protoDirs` and `outPath` first. But perhaps you have further ideas.